src/services/projects.js

```diff
@@ -383,8 +390,21 @@ module.exports = class ProjectsHelper {
 					}
 				}
 
+				if (fieldData && Object.keys(entityType.validations).includes('max_length')) {
```

Comment:
@adithyadinesh0412 The input field that allows for the creator to add a note to the reviewer will allow for an alphanumeric input.